src/test/java/org/springframework/cloud/deployer/spi/cloudfoundry/CloudFoundryAppDeployerTests.java

```diff
@@ -137,21 +140,22 @@ public void shouldNotNamespaceTheDeploymentIdWhenNoGroupIsUsed() throws Interrup
 	public void moveAppPropertiesToSAJ() throws InterruptedException, IOException {
 
 		// given
-		CloudFoundryDeployerProperties properties = new CloudFoundryDeployerProperties();
+		CloudFoundryConnectionProperties properties = new CloudFoundryConnectionProperties();
+		CloudFoundryDeploymentProperties deploymentProperties = new CloudFoundryDeploymentProperties();
 
 		// Define the deployment properties for the app
-		Map<String, String> deploymentProperties = new HashMap<>();
+		Map<String, String> appDeploymentProperties = new HashMap<>();
 
 		final String fooKey = "spring.cloud.foo";
 		final String fooVal = "this should end up in SPRING_APPLICATION_JSON";
 
 		final String barKey = "another.cloud.bar";
 		final String barVal = "this should too";
 
-		deploymentProperties.put(fooKey, fooVal);
-		deploymentProperties.put(barKey, barVal);
+		appDeploymentProperties.put(fooKey, fooVal);
+		appDeploymentProperties.put(barKey, barVal);
 
-		deployer = new CloudFoundryAppDeployer(properties, operations, client, deploymentCustomizer);
+		deployer = new CloudFoundryAppDeployer(properties, deploymentProperties, operations, client, deploymentCustomizer);
```

Comment:
No, I believe this is correct (check the types). `appDeploymentProperties ` are passed later below.